common/generate_bundle_base

```diff
@@ -50,7 +50,7 @@ fi
 
 if [ -n "${MASTER_OPTS[MODEL_CONSTRAINTS]}" ]; then
     juju set-model-constraints ${MASTER_OPTS[MODEL_CONSTRAINTS]}
-else
+elif ! ${MASTER_OPTS[HYPERCONVERGED_DEPLOYMENT]}; then
```

Comment:
We are not setting any constraints in the `else` case? I am not saying that we
should, but that's a change in behavior.

Reply:
there are no constraints to set are there?

Reply:
I don't think so. I just wanted to make sure I am not missing anything.